### PR TITLE
Img gallery

### PR DESCRIPTION
**Pull request by LindseyWeiskopf**
Created an image gallery page to go along with the portfolio

### CODE REVIEW

File: portfolio/src/main/webapp/gallery.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <a href="images/image-10.jpg"><img src="images/image-10.jpg"/></a>
+        <a href="images/image-11.jpg"><img src="images/image-11.jpg"/></a>
+        <a href="images/image-12.jpg"><img src="images/image-12.jpg"/></a>
+        <a href="images/image-13.jpg"><img src="images/image-13.jpg"/></a>
+        <a href="images/image-14.jpg"><img src="images/image-14.jpg"/></a>
+        <a href="images/image-15.JPG"><img src="images/image-15.JPG"/></a>
+        <a href="images/image-16.jpg"><img src="images/image-16.jpg"/></a>
+      </div>
+    </div> 
+  </body>
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/gallery.html:31]: Can you add a closing html tag here? As of the HTML5 specification, you can also omit it. I'll leave it up to you.

ref: https://google.github.io/styleguide/htmlcssguide.html , https://html.spec.whatwg.org/multipage/syntax.html#syntax-tag-omission

File: portfolio/src/main/webapp/gallery.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  </head>
+  <body>
+    <div id="gallery">
+      <h1>Here are some pictures that I like</h1>
+
+      <div class="image-thumbnails">
+        <a href="images/image-1.JPG"><img src="images/image-1.JPG"/></a>
+        
+        <a href="images/image-2.JPG"><img src="images/image-2.JPG"/></a>
+        <a href="images/image-3.jpg"><img src="images/image-3.jpg"/></a>
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/gallery.html:15]: nit: the extensions here are inconsistent. Some are ".JPG" while others are ".jpg".

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,41 @@
...
+      <h1>Hey, I'm Lindsey.</h1>
+      <h2>It's nice to meet you.</h2>
+      <p>This is me.</p>
+      <img src="images/profilepic.JPG"/>
+    </div>
+
+    <div id="projects">
+    <p>Here are some of the projects I've been working on:</p>
+    <ul>
+      <li>CU@Home - an augmented reality snapchat filter that lets you
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:20]: Please prefer a newline for each list element. https://google.github.io/styleguide/htmlcssguide.html#General_Formatting

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,41 @@
...
+      <li>Fractals with Python Turtle - An interactive program that lets
+      you create different fractals of differeing orders</li>
+      <li>Java SpellChecker - checks to see if your word is an a given
+      dictionary and gives you suggestions if it's not</li>
+    </ul>
+    </div>
+
+    <div id="links">
+	    <p>Find me on <a href="https://github.com/LindseyWeiskopf">GitHub</a> and <a href="https://www.linkedin.com/in/lindsey-weiskopf">LinkedIn</a>.</p>
+    </div>
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:31]: nit: Please use indentations of 2 spaces at a time. 

https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -14,10 +14,10 @@
 
 /**
  * Adds a random greeting to the page.
- */
+**/ 
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I am left handed', 'I go to Barnard College in NYC', 'Slow down and enjor the simple pleasures of life', 'Do not count the days, make the days count'];
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/script.js:20]: You can shorten this with s/go to/attend

Comment (shaquille-hall) [inline portfolio/src/main/webapp/script.js:20]: I think you meant "enjoy" here.

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -26,3 +26,23 @@ function addRandomGreeting() {
...
+
+  // Pick a random fact
+  const fact = facts[Math.floor(Math.random() * facts.length)];
+
+  // Add it to the page
+  const factContainer = document.getElementbyId('greeting-container');
+  factContainer.innerText=fact;
+}
+
+*/
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/script.js:48]: This is commented out. If you're not planning to delete it, it's a good practice to delete it.

File: portfolio/src/main/webapp/gallery.html (hunk 1 above)
Comment (LindseyWeiskopf) [reply portfolio/src/main/webapp/gallery.html:29]: Done!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (LindseyWeiskopf) [reply portfolio/src/main/webapp/index.html:20]: Got it.

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (LindseyWeiskopf) [reply portfolio/src/main/webapp/script.js]: Done!

Comment (LindseyWeiskopf) [reply portfolio/src/main/webapp/script.js]: yes, thanks!

**Pull request merged**